content/800-data-platform/200-projects/100-create.mdx

```diff
@@ -0,0 +1,152 @@
+
+After you create your project, you can optionally link your Prisma schema (if you push it to a GitHub repository) so that we can keep your project in sync with its latest version and enable you and your team members to work with the [Data Browser](/data-platform/data-browser), [Query Console](/data-platform/query-console), and [Schema Viewer](/data-platform/schema-viewer).
+
+Later, if you want to enable connection pooling and load balancing of database connections with the Data Proxy, you can copy the Data Proxy connection string for the project.
+
+</TopBlock>
+
+## Create a project
+
+When you create a project, you only need a database connection string.
+
+### Prerequisites
+
+Copy the database connection string for your project. The platform uses the connection string only to read and introspect the database.
+
+<Admonition>
+
+**Note**<br /><br />
+
+* Use a [Prisma ORM-supported database](/reference/database-reference/supported-databases).
+* The connection string must include `user` and `password`
+* Use the connection string format below.
+    ```code no-copy
+    <db-type>://<user>:<password>@<hostname>:<port>/<db-name>
+    ```
+
+</Admonition>
+
+### Steps
+
+1. After you [sign up for the Prisma Data Platform](/data-platform/account#sign-up-for-the-prisma-data-platform), go to the [_Projects_](https://cloud.prisma.io/projects) page.
+2. Click **New Project**.
+3. In **Connection string**, paste the database connection string.
+4. (Optional) From **Static IPs**, select **Enabled** if your database is behind a firewall and you can only configure external access from specific IP addresses. Copy the IP addresses and add them to the allowlist of your database.
+    <Admonition>
+
+    **Note**<br /><br />
+    For more information, see [Static egress IP address](/data-platform/static-ips). 
+    
+    </Admonition>
+5. From **Choose a Data Proxy region**, select the geographic location for the Data Proxy.
+    <Admonition>
+    
+    **Note**<br /><br />
+    Select a Data Proxy location that is close to the geographic location of your database.<br /><br />
+    If you select a location that is remote from your database, then when your application sends requests to your database through the Data Proxy, you might notice some delay in the responses.
+    </Admonition>
+6. Click **Create project**.
+    <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1669396285/docs/pdp/create-manage-projects/create-project-01-paste-connection-string-select-data-proxy-region-enable-static-ips_p5wauf.png"
+        alt="Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs"
+        width="500px"
+    />
+
+### Result
+
+The platform creates your project and assigns it a random name, such as **Grey&nbsp;fish&nbsp;<random-code\>**, where **<random-code\>** is an auto-generated code.
+<br />
```

Comment:
Shall we add a mention that you can later change this name in project settings?

Reply:
what a team @sampolahtinen @zachtil <3